### ISSUE TITLE
accessibility: hotgraphic pop back and next buttons broken

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2707]
### Subject
The tabbing is processed before the active item is changed.
https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/master/js/hotgraphicPopupView.js#L66-L67
These lines should be swapped.

When you click next, the focus isn't moving to the next item title because it is still aria-hidden.

**Comment by moloko** [comment]
I think this needs to be fixed for the Adapt v4 compatible version of hotgraphic as well...
